presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkQueryExecutionFactory.java

```diff
@@ -434,7 +438,7 @@ public IPrestoSparkQueryExecution create(
                                 warningCollector));
 
                 log.info(textDistributedPlan(fragmentedPlan, metadata.getFunctionAndTypeManager(), session, true));
-                fragmentedPlan = configureOutputPartitioning(session, fragmentedPlan);
+                fragmentedPlan = configureOutputPartitioning(session, fragmentedPlan, planAndMore.getPhysicalResourceSettings(), planAndMore.getPhysicalResourceSettings().isValid());
```

Comment:
`planAndMore.getPhysicalResourceSettings()` is already being passed to `configureOutputPartitioning`. So, passing `planAndMore.getPhysicalResourceSettings().isValid()` separately is not needed. You can directly call `isValid()` on the `PhysicalResourceSettings` object inside the method.

Reply:
Yeah, definitely redundant passing the isValid() flag. Updated.